sp7-stats/index.php

```
@@ -166,223 +166,258 @@ class="form-control"><?=$result_2?></select>
+        let count = '<?=$times?>';
+        let search_query = '<?=$search_query?>';
 
-	const first_day = '<?=$first_day?>';
-	const last_day = '<?=$last_day?>';
-	const view = '<?=$view?>';
+        const first_day = '<?=$first_day?>';
+        const last_day = '<?=$last_day?>';
+        const view = '<?=$view?>';
 
-	const link = '<?=LINK?>?'; <?php
+        const link = '<?=LINK?>?'; <?php
```

PHPStan (2.2.2)

[error] Constant LINK not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)